.github/workflows/build.yml

```
@@ -25,8 +30,22 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_PULL_USERNAME }}
           password: ${{ secrets.DOCKER_PULL_PASSWORD }}
-      - name: Build image variant "${{ matrix.variant }}"
-        run: cd ${{ matrix.variant }} && docker buildx build ./ -t ${{ matrix.docker-tag }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: --debug
+
+      - name: Build and push image for "${{ matrix.variant }}"
+        uses: docker/build-push-action@v3
+        with:
+          context: ${{ matrix.variant }}
+          platforms: "linux/arm/v6,linux/arm/v7,linux/s390x,linux/ppc64le,linux/386,linux/amd64,linux/arm64/v8"
```

```suggestion
          platforms: "linux/arm/v6,linux/arm/v7,linux/s390x,linux/ppc64le,linux/386,linux/amd64"
```

`linux/arm64/v8` is not supported by composer or alpine PHP images

---

removing the v8 now

---

It's possible I am mistaken on this because they might be the same between `linux/arm64/v8` and `linux/arm64`.
Did builds fail on it ?